### ISSUE TITLE
Collection stats of a set can get missing squares

### ISSUE THREAD

**Issue by Porkepix** [Manuel-777/MTG-Arena-Tool#779]
See the joined screenshot for all missing squares for example on Ravnica Alligiance. My guess is that it's the same bug as #773: I think internally counter goes over 4 and that the UI does not know what to do with a counter above 5.

![new_cards_bug](https://user-images.githubusercontent.com/709500/69920876-63786400-148d-11ea-96cc-c3a47e7bb908.PNG)

**Comment by Manuel-777** [comment]
The number should still show up, also its impossible for the counter to go beyond 4, since that is read from your collection.
Could you share your user config file + an output log at mtgatool@gmail.com ?

**Comment by Porkepix** [comment]
Sure, but I have no idea how to get logs from a software on Windows.
I also went in the application's directory and found a `logs` directory empty. And an `action_logs` one with a ton of files.

**Comment by Manuel-777** [comment]
In [TROUBLESHOOTING](https://github.com/Manuel-777/MTG-Arena-Tool/blob/master/TROUBLESHOOTING.md) we detail where these can be found.

**Comment by Porkepix** [comment]
Yup, I already looked there, but it does refer only to MTGA logs, not MTG Arena Logs.
If you refer to MTGA logs, aside from containing many things unrelated to MTG Arena Tools, it's also quite big (110+MB currently).
